tests/test_kernels.py

```diff
@@ -0,0 +1,89 @@
+"""Kernel tests."""
+
+from typing import Any
+
+import numpy as np
+import torch
+from attrs import asdict, has
+from hypothesis import given
+
+from baybe.kernels.base import Kernel
+from baybe.kernels.composite import AdditiveKernel, ProductKernel, ScaleKernel
+
+from .hypothesis_strategies.kernels import kernels
+
+# TODO: Consider deprecating these attribute names to avoid inconsistencies
+_RENAME_DICT: dict[str, str] = {
+    "base_kernels": "kernels",
+}
+"""Dictionary for resolving name differences between BayBE and GPyTorch attributes."""
+
+
+def validate_kernel_components(obj: Any, mapped: Any, **kwargs) -> None:
```

Comment:
Would appreciate to include some information that this is necessary for GPyTorch kernels: Although we only have these ones in the moment, I'd like if that information was in here explicitly (either by changing the function's name or adjusting the docstring)

Reply:
In particular since this information is currently implicitly contained in the `Args:` section, we should make this more prominent imo.

Reply:
done